tox.ini

```
@@ -3,7 +3,9 @@ envlist =
     py{37,38,39}-django22,
     py{37,38,39}-django{30,31},
     py{37,38,39,310}-django32,
-    py{38,39,310}-django{40,41,master},
+    py{38,39,310}-django40,
+    py{38,39,310,311}-django{41,42},
+    py{310,311}-main,
```

```suggestion
    py{310,311}-djangomain,
```
@kiendang should it actually be `djangomain`?

---

Yup you're right.